kpops/cli/pipeline_config.py

```diff
@@ -107,8 +107,8 @@ def customise_sources(
         ):
```

Comment:
add the return type annotation please

Reply:
Done, is it correct?

Reply:
and you also need to change the return value of the `yaml_config_settings_source` function 😛

Reply:
This throws an error:

```
Expression of type "tuple[SettingsSourceCallable, SettingsSourceCallable, (settings: PipelineConfig) -> (dict[Unknown, Unknown] | list[Unknown]), SettingsSourceCallable]" cannot be assigned to return type "tuple[SettingsSourceCallable, ...]"
  Type "(settings: PipelineConfig) -> (dict[Unknown, Unknown] | list[Unknown])" cannot be assigned to type "SettingsSourceCallable"
    Parameter 1: type "BaseSettings" cannot be assigned to type "PipelineConfig"
      "BaseSettings" is incompatible with "PipelineConfig"
    Function return type "dict[Unknown, Unknown] | list[Unknown]" is incompatible with type "Dict[str, Any]"
      Type "dict[Unknown, Unknown] | list[Unknown]" cannot be assigned to type "Dict[str, Any]"
        "list[Unknown]" is incompatible with "Dict[str, Any]"Pylance[reportGeneralTypeIssues](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportGeneralTypeIssues)
```

Reply:
ah, I never saw your answers from 30 minutes ago, should've refreshed the page before answering :|